test/unit-tests/mobile/android/logcat-helper.ts

```diff
@@ -0,0 +1,125 @@
+import { LogcatHelper } from "../../../../mobile/android/logcat-helper";
+import { Yok } from "../../../../yok";
+import { assert } from "chai";
+import * as path from "path";
+import * as childProcess from "child_process";
+
+class ChildProcessStub {
+	public static methodCallCount = 0;
+	private isWin = /^win/.test(process.platform);
+
+	public spawn(command: string, args?: string[], options?: any): childProcess.ChildProcess {
+		ChildProcessStub.methodCallCount++;
+		let pathToExecutable = "";
+		if (this.isWin) {
+			pathToExecutable = "getcontent.bat";
```

Comment:
As far as I tested it out there is no common command to use between unix machines and windows to print out a file content to the stdout.
`type` is windows specific and `cat` is unix speciffic -> [read more](http://www.yolinux.com/TUTORIALS/unix_for_dos_users.html)

Reply:
I fully agree with this, my point is that you do not need to use `getcontent` executable. Just set the `pathToExecutable` to `type` in case you are on Windows and `cat` otherwise.